Makefile

```
@@ -237,7 +237,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: image-local-build
 image-local-build:
 	BUILDER=$(shell $(DOCKER_BUILDX_CMD) create --use)
-	$(MAKE) image-build PUSH=$(PUSH) IMAGE_BUILD_EXTRA_OPTS=$(IMAGE_BUILD_EXTRA_OPTS)
+	$(MAKE) image-build PUSH=$(PUSH) IMAGE_BUILD_EXTRA_OPTS="$(IMAGE_BUILD_EXTRA_OPTS)"
```

```suggestion
	$(MAKE) image-build PUSH=$(PUSH) $(IMAGE_BUILD_EXTRA_OPTS)
```

Maybe like this? For consistency with other places:

 https://github.com/kubernetes-sigs/kueue/blob/4457de2a5ead5bea31f115be1f333f7ab2d33138/Makefile#L260.

---

Does not seem to work:

```sh
> make image-local-push
BUILDER=agitated_swartz
make image-build --push 
make: unrecognized option '--push'
Usage: make [options] [target] ...
```
Also, from consistency POV, should it pass like this: `PUSH=$(PUSH) IMAGE_BUILD_EXTRA_OPTS="$(IMAGE_BUILD_EXTRA_OPTS)" $(MAKE) image-build`

This is what we do for LD_FLAGS which are another good example of a param with spaces: https://github.com/kubernetes-sigs/kueue/blob/638020cb863e9151890ca665fde1dc2873b47cb5/Makefile#L348

---

Ah, I see. Thanks for double-checking!

---

yeah, pushed